pandas/core/reshape/merge.py

```diff
@@ -1015,6 +1015,43 @@ def _get_join_info(
         left_ax = self.left.axes[self.axis]
         right_ax = self.right.axes[self.axis]
 
+        if self.how in ["anti_left", "anti_right", "anti_full"]:
+            how_dict: dict[str, str] = {
```

Comment:
```suggestion
            how_dict = {
```

This should be inferrable from the assignment